packages/react-data-grid/src/common/types.ts

```diff
@@ -1,7 +1,18 @@
+import { HeaderRowType } from './enums';
+
+export type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
+
 export interface Column<T = unknown> {
   name: string;
   key: string;
+  rowType?: HeaderRowType;
```

Comment:
I think we should remove `rowType` as it not a column prop

Reply:
Removed.